backend/api.go

```diff
@@ -0,0 +1,147 @@
+package main
```

Comment:
I'd like you to work on domain naming - firstly, think of package name that fits the domain use case of this code. Then, create a new directory, for example `internal/database` and move this file there - with package name changed to `database`. Also, the filename could be more informative as `api` is too general.

See:
- https://golang.org/doc/effective_go#package-names

Reply:
Added to the next sprint.